configure/os/CONFIG.Common.linux-riscv64

```diff
@@ -0,0 +1,21 @@
+# CONFIG.Common.linux-x86_64
+#
+# Definitions for linux-x86_64 target builds
+# Sites may override these definitions in CONFIG_SITE.Common.linux-x86_64
+#-------------------------------------------------------
+
+# Include definitions common to all linux targets
+include $(CONFIG)/os/CONFIG.Common.linuxCommon
+
+ARCH_CLASS = riscv64
+
+ARCH_DEP_CFLAGS = $(GNU_TUNE_CFLAGS)
+ARCH_DEP_CPPFLAGS += -D_RISCV64_
```

Comment:
Please remove the `-D_RISCV64_` definition.  Historically, Base provided its own ID macros for some targets.  However, these days we prefer to depend on compiler pre-defined macros.

eg. with GCC, query pre-defined macros by running `riscv64-linux-gnu-gcc -dM -x c`.

Comparing the outputs for riscv 32 vs. 64 from [godbolt.org](https://godbolt.org/#g:!((g:!((g:!((h:codeEditor,i:(filename:'1',fontScale:14,fontUsePx:'0',j:1,lang:c%2B%2B,source:'//+Type+your+code+here,+or+load+an+example.%0Aint+square(int+num)+%7B%0A++++return+num+*+num%3B%0A%7D'),l:'5',n:'1',o:'C%2B%2B+source+%231',t:'0')),k:33.333333333333336,l:'4',n:'0',o:'',s:0,t:'0'),(g:!((h:compiler,i:(compiler:rv64-gcc1420,filters:(b:'0',binary:'1',binaryObject:'1',commentOnly:'0',debugCalls:'1',demangle:'0',directives:'0',execute:'1',intel:'0',libraryCode:'0',trim:'1',verboseDemangling:'0'),flagsViewOpen:'1',fontScale:14,fontUsePx:'0',j:1,lang:c%2B%2B,libs:!(),options:'-dM+-x+c',overrides:!(),selection:(endColumn:19,endLineNumber:15,positionColumn:19,positionLineNumber:15,selectionStartColumn:19,selectionStartLineNumber:15,startColumn:19,startLineNumber:15),source:1),l:'5',n:'0',o:'+RISC-V+(64-bits)+gcc+14.2.0+(Editor+%231)',t:'0')),k:33.333333333333336,l:'4',n:'0',o:'',s:0,t:'0'),(g:!((h:pp,i:(clang-format:'1',editorid:1,filter-headers:'0',fontScale:14,fontUsePx:'0',j:1,treeid:0),l:'5',n:'0',o:'Preprocessor+Output+RISC-V+(64-bits)+gcc+14.2.0+(Editor+%231,+Compiler+%231)',t:'0')),k:33.33333333333333,l:'4',n:'0',o:'',s:0,t:'0')),l:'2',n:'0',o:'',t:'0')),version:4) leads me towards ID macros `#define __riscv` and `#define __SIZEOF_POINTER__`.

eg. testing the most specific case:

```c
#if defined(__linux__) && defined(__GNUC__) && defined(__riscv) && __SIZEOF_POINTER__==8
/* GCC compiling for Linux running on RISC-V 64 */
#endif
```

Reply:
done